javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,22 @@
+var fullName;
+var isCool;
+var totalPets;
+var unknown;
+var never;
+
+fullName = 'Markus Doan-Le';
+isCool = false;
+totalPets = 2;
+unknown = null;
```

Comment:
This is not necessary in JavaScript, you can declare a variable and assign a value on the same line.